website/api_versioned_docs/version-1.2/zkvm/precompiles.md

```diff
@@ -33,6 +34,19 @@ git = "https://github.com/risc0/RustCrypto-hashes"
 tag = "sha2-v0.10.6-risczero.0"
 ```
 
+Make sure that your dependency gives the same version of the crate as listed in
+the patch tag. In this example, the dependency would be
+```toml
+[dependencies]
+sha2 = "0.10.6"
+```
+
+In some situations, particularly for crates you've used before with patch
+version updates, you may need to regenerate your `lock` file. For example:
```

Comment:
```suggestion
In some situations, for example when a patch is used indirectly, you may
need to update the version specifically to update your lockfile. For example:
```

Personally unclear about this wording, plus this is the main reason why you might need to manage the patch version like this I think.